src/Foundation.hs

```diff
@@ -137,9 +138,9 @@ instance Yesod App where
 
     -- Store session data on the client in encrypted cookies,
     -- default session idle timeout is 120 minutes
-    makeSessionBackend _ = Just <$> defaultClientSessionBackend
+    makeSessionBackend _ = Just <$> envClientSessionBackend
         120    -- timeout in minutes
-        "config/client_session_key.aes"
+        "PURSUIT_CLIENT_SESSION"
```

Comment:
`defaultClientSessionBackend` creates `config/client_session_key.aes` in the installation directory on the first run. This doesn't play well where the installation directory is immutable.
`envClientSessionBackend` allows passing the client session key as an environment variable.

While on subject, I think `PURSUIT_CLIENT_SESSION`  should be changed to `PURSUIT_CLIENT_SESSION_KEY`